app/resources/generated_resources_en-GB.xtb

```diff
@@ -2050,6 +2070,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
 <translation id="2650609753175229582"><ph name="URL"/> wants to play protected content. Your device's identity will be verified by Brave Software and may be accessed by this site.</translation>
 <translation id="1746402432151920942">Protected media identifier</translation>
 <translation id="1322046419516468189">View and manage saved passwords in your <ph name="SAVED_PASSWORDS_STORE"/></translation>
+<translation id="3312599731068160560">https://passwords.google.com</translation>
```

Comment:
Where is this string showing up?

Reply:
`chrome/browser/android/password_ui_view_android.cc:221`